app/main.py

```
@@ -11,13 +11,16 @@
 from fastapi import FastAPI
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.middleware.gzip import GZipMiddleware
+from fastapi.middleware.trustedhost import TrustedHostMiddleware
 from prometheus_fastapi_instrumentator import Instrumentator
 from sentry_sdk.integrations.asgi import SentryAsgiMiddleware
```

Ruff (0.11.9)

`sentry_sdk.integrations.asgi.SentryAsgiMiddleware` imported but unused

Remove unused import: `sentry_sdk.integrations.asgi.SentryAsgiMiddleware`

(F401)